Source/Controllers/Other/SQLiteHistoryManager.swift

```diff
@@ -351,26 +347,23 @@ typealias SASchemaBuilder = (_ db: FMDatabase, _ schemaVersion: Int) -> Void
     ///   - statement: Int - the command that failed
     ///   - db: FMDatabase - the FMDatabase instance
     /// - Returns: nothing, should crash
-    func failedAt(statement: Int, db: FMDatabase) {
-        let lastErrorCode = db.lastErrorCode()
-        let lastErrorMessage = db.lastErrorMessage()
-        db.rollback()
-        assert(0 != 0, "Migration statement \(statement) failed, code \(lastErrorCode): \(lastErrorMessage)")
+    func failedAt(_ error: Error) {
```

Comment:
we should probably update the name and docu - `failed(error: Error)` for example